### ISSUE TITLE
Release pygloo to pypi

### ISSUE THREAD

**Issue by jovany-wang** [ray-project/pygloo#17]

**Comment by ZeldaHuang** [comment]
Hi, I run`pip install pygloo` and its not released in pypi, any plan to release it?

**Comment by jovany-wang** [comment]
Yes. The manylinux environment is fixed at https://github.com/ray-project/pygloo/pull/30
But right now, I'm acquiring the releasing permission from others.

**Comment by jovany-wang** [comment]
@ZeldaHuang  If you need this right now, you could build the wheels following this script:
https://github.com/ray-project/pygloo/blob/main/.github/workflows/manylinux_basic.yml#L20-L28

And then, run build_wheel.sh

**Comment by jovany-wang** [comment]
[0.2.1](https://pypi.org/project/pygloo/0.2.1/) was released from out team.
